charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-daemonset.yaml

```diff
@@ -35,11 +35,11 @@ spec:
               fieldPath: spec.nodeName
         resources:
           limits:
-            cpu: 100m
-            memory: 100Mi
+            cpu: 300m
+            memory: 400Mi
           requests:
-            cpu: 20m
-            memory: 40Mi
+            cpu: 200m
+            memory: 200Mi
```

Comment:
I am fine with the increased limits, but why are you increasing the requests to so high values?

Reply:
Hm, indeed, it seems not necessary. The old request values were quite OK